src/main/java/io/jenkins/plugins/orka/client/OrkaClient.java

```diff
@@ -156,8 +163,11 @@ private Builder getAuthenticatedBuilder(String url) throws IOException {
     }
 
     private String executeCall(Request request) throws IOException {
+        logger.log(Level.INFO,"Executing request to Orka API: " + '/' + request.method()  + ' ' + request.url());
+
         try (Response response = client.newCall(request).execute()) {
-            return response.body().string();
+            String stringResponse =  response.body().string();
```

Comment:
You do not need this variable

Reply:
I was looking at one logging interceptor for the OkHttp library and this was a leftover. :)